### PR TITLE
refactor(swift 5/code): added Swift 5 compatibility for Xcode 10.2 and iOS 12.2

### PR DESCRIPTION
**Pull request by Pomanks**
removed redundant code and formatted everything with SwiftFormat (e.g. removed whitespaces, added space between colon and variables, …).

removed force casts and unnecessary type casting.

### CODE REVIEW

File: Spring/KeyboardLayoutConstraint.swift [hunk 1]
```diff
@@ -23,88 +23,87 @@
...
+
+        deinit {
+            NotificationCenter.default.removeObserver(self)
+        }
+
+        // MARK: Notification
+
+        @objc func keyboardWillShowNotification(_ notification: Notification) {
+            if let userInfo = notification.userInfo {
+                if let frameValue = userInfo[UIResponder.keyboardFrameEndUserInfoKey] as? NSValue {
```
Comment (musbaalbaki) [inline Spring/KeyboardLayoutConstraint.swift]: 'keyboardFrameEndUserInfoKey' has been renamed to 'UIKeyboardFrameEndUserInfoKey'

Comment (Pomanks) [reply Spring/KeyboardLayoutConstraint.swift]: Which target are you building when this warning appears ?
I just build again and I don't have anything.. 🤔

**Comment by MengTo** [comment]
Hi, thank you for this. Do you mind pushing to swift5 branch so we can test from there first?

**Comment by Pomanks** [comment]
Couldn't push directly to your swift5 branch, changed pull request branch instead (from master to swift5)

**Pull request merged**